packages/orf/orf.1.0.0/opam

```diff
@@ -0,0 +1,60 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/orf"
+bug-reports: "https://github.com/UnixJunkie/orf/issues"
+dev-repo: "git+https://github.com/UnixJunkie/orf.git"
+license: "LGPL w/ linking exception"
```

Comment:
Would
```suggestion
license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
```
Be correct? I could not find the license file in the repo (did not search too hard though)